cmd/osnadmin/main.go

```diff
@@ -50,7 +50,7 @@ func executeForArgs(args []string) (output string, exit int, err error) {
 
 	join := channel.Command("join", "Join an Ordering Service Node (OSN) to a channel. If the channel does not yet exist, it will be created.")
 	joinChannelID := join.Flag("channel-id", "Channel ID").Short('c').Required().String()
-	configBlockPath := join.Flag("config-block", "Path to the file containing the config block").Short('b').Required().String()
+	configBlockPath := join.Flag("config-block", "Path to the file containing the config block. This can be either the genesis block or a config block for the channel.").Short('b').Required().String()
```

Comment:
Is this to address another comment? The genesis block is a config block, it's just the first one.  Really, this should be the latest config block, or, a config block which contains the current set consenters and corresponding TLS CAs (as required to replicate the channel).

Reply:
Not sure how much we can/should bake into a one/two-liner. Maybe switch back towards the original text with a minor clarification: "Path to the file containing a config block for the channel"?

Reply:
Perhaps something like: "Path to the file containing an up to date config block for the channel."?  Maybe that creates more confusion than clarity.  My biggest concern was that adding the longer statement about genesis or config was verbose without adding info.

Reply:
Switched to "up-to-date config block".